src/tracks/InstrumentTrack.cpp

```diff
@@ -771,7 +771,7 @@ void InstrumentTrack::saveTrackSpecificSettings( QDomDocument& doc, QDomElement
 
 void InstrumentTrack::loadTrackSpecificSettings( const QDomElement & thisElement )
 {
-	silenceAllNotes( true );
+	silenceAllNotes(!(m_previewMode && m_instrument && m_instrument->nodeName() == getInstrumentName(thisElement)));
```

Comment:
I think we may extract the conditions in parentheses to a variable like `reuseInstrument` to make the meaning clear and reuse it below.